JapaneseFilter/sections/specific.txt

```diff
@@ -2140,6 +2140,7 @@ itmedia.co.jp###cmsSubPR
 itmedia.co.jp###ITN
 itmedia.co.jp###ITT
 itmedia.co.jp###cmsBodyMain > div:not([class]) > div[style="text-align:center;margin-bottom:20px;"]:only-child
+itmedia.co.jp##.colBoxAd
```

Comment:
Not an ad. Please move this rule to AdGuard Annoyances - self-promo.txt.

Reply:
I see